tests/data/special_character/definitions/variable/variables.yaml

```diff
@@ -0,0 +1,9 @@
+- Primary Energy:
```

Comment:
I would shorten this file to include maybe only the variable with the hidden character. Also I don't think the tags are needed.